app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsContributionsHeaderView.kt

```diff
@@ -0,0 +1,77 @@
+package org.wikipedia.suggestededits
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import kotlinx.android.synthetic.main.view_contributions_header.view.*
+import org.wikipedia.R
+import org.wikipedia.suggestededits.Contribution.Companion.EDIT_TYPE_IMAGE_CAPTION
+import org.wikipedia.suggestededits.Contribution.Companion.EDIT_TYPE_IMAGE_TAG
+
+class SuggestedEditsContributionsHeaderView constructor(context: Context, attrs: AttributeSet? = null) : LinearLayout(context, attrs), SuggestedEditsTypeItem.Callback {
+    interface Callback {
+        fun onTypeItemClick(editType: Int)
+    }
+
+    var callback: Callback? = null
+    private var filterViews = ArrayList<SuggestedEditsTypeItem>()
+
+    init {
+        View.inflate(context, R.layout.view_contributions_header, this)
+        orientation = VERTICAL
+        layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, ViewGroup.LayoutParams.WRAP_CONTENT)
```

Comment:
Looks like the `R.layout.view_contributions_header` is a `LinearLayout`, should we change the view to use `merge`?

Reply:
Yep, forgot to finish that one.